### ISSUE TITLE
Support animated-WebP for FrescoImage

### ISSUE THREAD

**Issue by Notsfsssf** [skydoves/Landscapist#44]
Hi,
Fresco's animated-webp support is impl with `SimpleDraweeView`
Is there any way to support dynamic pictures for `FrescoImage`?
https://frescolib.org/docs/animations.html
https://frescolib.org/docs/webp-support.html

**Comment by skydoves** [comment]
Hi @Notsfsssf,
Thanks for your issue!
Here is a new SNAPSHOT dependency for showing the webp.

```gradle
implementation "com.github.skydoves:landscapist-fresco-websupport:1.3.4-SNAPSHOT"
```

You can use `FrescoWebImage` for showing a webp file.

```kotlin
FrescoWebImage(
      controllerBuilder = Fresco.newDraweeControllerBuilder()
        .setUri("asset:///test.webp")
        .setAutoPlayAnimations(true),
      modifier = Modifier.size(250.dp)
    )
```

**Comment by skydoves** [comment]
Released a new stable `1.3.4`.
Thank you for your issue!
